### ISSUE TITLE
UNSUPPORTED_OPERATION, geAddress error

### ISSUE THREAD

**Issue by crypt0summer** [opengsn/gsn#960]
I'm testing on Polygon mumbai and testing from client side.
This is the part that makes error
``` 
const provider = new Web3.providers.HttpProvider(
    "https://polygon-mumbai.g.alchemy.com/v2/<MyAlchemyID>"
  );
const gsnProvider = RelayProvider.newProvider({
    provider: provider,
    config: {
      paymasterAddress: "0xe3e374701f06217a49EF12122841FbD427626085",
    },
  });
  await gsnProvider.init();
```

0xe3e374701f06217a49EF12122841FbD427626085 is my payMaster contract(SingleRecipientPyamster) that I deployed on mumbai, 
https://mumbai.polygonscan.com/address/0xe3e374701f06217a49EF12122841FbD427626085

Short error code :
```
  reason: 'unknown account #0',
  code: 'UNSUPPORTED_OPERATION',
  operation: 'getAddress'
```

Full error code : 
```
$ npx hardhat run scripts/interact.ts --network mumbai
Reading default client config for chainId 80001
Config response: {"version":"3.0.0-beta.6","networks":{"1":{"name":"mainnet","gsnConfig":{"tokenPaymasterDomainSeparators":{"0x6B175474E89094C44Da98b954EedeAC495271d0F":{"name":"Dai Stablecoin","version":"1","chainId":1,"verifyingContract":"0x6B175474E89094C44Da98b954EedeAC495271d0F"},"0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48":{"name":"USD Coin","version":"2","chainId":1,"verifyingContract":"0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"},"0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984":{"name":"Uniswap","chainId":1,"verifyingContract":"0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984"}}}},"5":{"name":"goerli","gsnConfig":{}}}}
interactor init start
Network supports Type 2 Transactions (EIP-1559). Checking RPC node 'eth_feeHistory' method
RPC node supports 'eth_feeHistory'. Initializing to Type 2 Transactions.
Error: unknown account #0 (operation="getAddress", code=UNSUPPORTED_OPERATION, version=providers/5.7.2)
    at Logger.makeError (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@ethersproject/logger/src.ts/index.ts:269:28)
    at Logger.throwError (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@ethersproject/logger/src.ts/index.ts:281:20)
    at /Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@ethersproject/providers/src.ts/json-rpc-provider.ts:204:24
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Contract.at (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@opengsn/common/src/LightTruffleContract.ts:74:31)
    at async ContractInteractor._createPaymaster (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@opengsn/common/src/ContractInteractor.ts:440:12)
    at async ContractInteractor._resolveDeploymentFromPaymaster (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@opengsn/common/src/ContractInteractor.ts:303:30)
    at async ContractInteractor._resolveDeployment (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@opengsn/common/src/ContractInteractor.ts:290:7)
    at async ContractInteractor.init (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@opengsn/common/src/ContractInteractor.ts:262:5)
    at async RelayClient._resolveDependencies (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@opengsn/provider/src/RelayClient.ts:637:7)
    at async RelayClient._initInternal (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@opengsn/provider/src/RelayClient.ts:147:25)
    at async RelayClient.init (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@opengsn/provider/src/RelayClient.ts:135:5)
    at async RelayProvider.init (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/node_modules/@opengsn/provider/src/RelayProvider.ts:94:5)
    at async main2 (/Users/sewon/Documents/Dev/Contract/PolygonNFT_GSN/scripts/interact.ts:33:3) {
  reason: 'unknown account #0',
  code: 'UNSUPPORTED_OPERATION',
  operation: 'getAddress'
```

What is wrong with my code?

**Comment by forshtat** [comment]
This appears to be the issue fixed by https://github.com/opengsn/gsn/pull/964 and released as [v3.0.0-beta.7](https://github.com/opengsn/gsn/releases/tag/v3.0.0-beta.7). If the issue still reproduces after update please post here, on Discord, or open a new issue.

**Comment by T-Damer** [comment]
Happens to me on `beta.8`, I also use Alchemy/Infura providers, they don't expose `getAddress` method
